homeassistant/components/switch/zha.py

```diff
@@ -15,19 +15,39 @@
 DEPENDENCIES = ['zha']
 
 
-def setup_platform(hass, config, add_devices, discovery_info=None):
-    """Set up Zigbee Home Automation switches."""
+@asyncio.coroutine
```

Comment:
We've migrated to `async`/`await` syntax. Please drop this decorator and prefix async methods with `async` like `async def async_setup_platform`. Replace any `yield from` with `await`.